CHANGELOG.md

```
@@ -4,6 +4,35 @@ Date format: (year/month/day)
 
 ## Change Log
 
+### Version 6.1.0 (2026/01/31)
+
+**Improvements**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)